src/main/java/uk/gov/digital/ho/hocs/search/api/HocsQueryBuilder.java

```diff
@@ -106,6 +106,26 @@ HocsQueryBuilder correspondentName(String correspondentName) {
         return this;
     }
 
+    HocsQueryBuilder correspondentNameNotMP(String correspondentNameNotMP) {
+        if (StringUtils.hasText(correspondentNameNotMP)) {
```

Comment:
Technically MP is a a subset here. Might be better to hardcode as "Rep" or representative?

Reply:
I've gone with "Member" to reflect the system type.